hack/util.sh

```diff
@@ -459,9 +459,7 @@ function install_registry() {
 readonly -f install_registry
 
 function wait_for_registry() {
-	local generation="$(oc get dc/docker-registry -o 'jsonpath={.metadata.generation}')"
-	local onereplicajs='{.status.observedGeneration},{.status.replicas},{.status.updatedReplicas},{.status.availableReplicas}'
-	wait_for_command "oc get dc/docker-registry -o 'jsonpath=${onereplicajs}' --config='${ADMIN_KUBECONFIG}' | grep '^${generation},1,1,1$'"  "$((5*TIME_MIN))"
+	wait_for_command "oc rollout status dc/docker-registry --request-timeout=1m"  "$((5*TIME_MIN))"
```

Comment:
@stevekuznetsov here is my reply at https://github.com/openshift/origin/pull/11410#issuecomment-254574288

We don't even need a separate helper for waiting for dc/xxxx to be ready now. Let's just use `oc rollout status` (waits for a complete status on top of a synced config (dc.generation <= dc.status.observedGeneration).